### PR TITLE
Make up more immune to app code crashes

### PR DESCRIPTION
**Pull request by vladistan**
App is more stable when following is done
- Run node in server mode
- Restart failed containers

### CODE REVIEW

**Comment by neka-io** [comment]
@vladistan what do you think about using [Forever](https://github.com/foreverjs/forever) to keep the Node server running?

**Comment by vladistan** [comment]
@noonkay this change is does essentially the same thing.  Some users still get 502 while it restarts.  But it its only for a spilt second.

File: searchtool/docker-compose.yaml [hunk 1]
```diff
@@ -24,6 +24,7 @@ services:
              - node_modules:/node_modules
          environment:
              - NODE_PATH=/usr/lib/node_modules:/node_modules
+         restart: always
```
Comment (vladistan) [inline searchtool/docker-compose.yaml:27]: @noonkay this does the same thing as forever

**Comment by neka-io** [comment]
Cool. Merging.

**Pull request merged**